src/Context/reducer.ts

```diff
@@ -89,10 +89,21 @@ const gameReducer = createReducer<GameState>({
         ...state,
         playerId,
     }),
-    [USER_EVENTS.REJOIN_GAME]: (state, gameState: Partial<GameState>) => ({
+    [USER_EVENTS.REJOIN_GAME]: (state, gameState: RejoinGameParams): GameState => {
+        const { gameStarted, allBetsIn, ...rest } = gameState;
+
+        return {
             ...state,
-            ...gameState,
-    }),
+            ...rest,
+            stage: (
+                gameStarted ?
```

Comment:
I think using more than 1 ternary operator in a line gets difficult to read. can we split this up?

Reply:
https://github.com/creisle/quarantined_wizzards/pull/30/commits/ac31a8b49e9460059bf1da6a4b38a800b1aadb28